Tests/Core/PrettyTests.swift

```diff
@@ -132,6 +132,19 @@ class PrettyTests: XCTestCase {
             when(true,  then: #"["mike": Cat(id: "mike", name: Optional("ポチ")), "tama": Cat(id: "tama", name: Optional("タマ"))]"#),
         ])
     }
+    
+    ///
+    /// Tuple
+    ///
+    func testString_Tuple() {
+        let tuple = (1, ("one", URL(string: "https://www.example.com/")!))
```

Comment:
Labeled tuple (e.g. `(one: 1, two: "two")`) is not support in this PR?

Reply:
I'll try it.

Reply:
Label supported in d43f963